pagefaultgames/pokerogue
# [Sprite] Fix Courtney eye whites

## What are the changes the user will see?
Magma Admin Courtney's sprite now has white sclera, as well as a few other optimisations.

## Why am I making these changes?
Reported [here](https://discord.com/channels/1125469663833370665/1277005388692066414).

Replaced transparent sclera regions with opaque white pixels.
Erased a white pixel at the top edge of the sprite.
Replaced several near-black palette entries with the standard 0x101010.
Converted the sprite to PNG-8 indexed bit depth.

## What are the changes from a developer perspective?
No mechanical changes. Replaced one .png with another.

### Screenshots/Videos
#### Before
![image](https://github.com/user-attachments/assets/a2e05db0-df2b-421d-94ce-1d1a6842b1bc)
#### After
![image](https://github.com/user-attachments/assets/52caab73-a4c1-4e0a-9eb6-5aeaec890abd)

## How to test the changes?
As far as I know, the optimal way to produce fresh legitimate results is to set STARTING_WAVE_OVERRIDE to 66 and hope for the best.

My results were achieved using this way, though naturally a PokeRogue SaVe should have the file ending `.prsv`.
[sessionData_Courtney.txt](https://github.com/user-attachments/files/16808266/sessionData_Courtney.txt)

Alternatively, renaming the files in `\images\trainers` can allow a more common encounter like a rival or a youngster to use Courtney's sprite .png and animation .json.

## Checklist
~~- [ ] Have I considered writing automated tests for the issue?~~
~~- [ ] If I have text, did I make it translatable and add a key in the English locale file(s)?~~

## Links:

- https://www.deviantart.com/icejkai/art/Team-Magma-Grunts-Trainer-Sprites-536151006
- https://www.deviantart.com/icejkai/art/Magma-Admin-Courtney-Trainer-Sprite-544510574
